influxql/ast.go

```diff
@@ -555,7 +592,7 @@ func (s *CreateRetentionPolicyStatement) String() string {
 
 // RequiredPrivileges returns the privilege required to execute a CreateRetentionPolicyStatement.
 func (s *CreateRetentionPolicyStatement) RequiredPrivileges() ExecutionPrivileges {
-	return ExecutionPrivileges{{Name: "", Privilege: AllPrivileges}}
+	return ExecutionPrivileges{{Admin: true, Name: "", Privilege: AllPrivileges}}
```

Comment:
Here's a proposal for permissions by GRANT type per opertation: https://docs.google.com/a/errplane.com/spreadsheets/d/1V1acjK8PS8QCQ3MagsvIMhT3giy8dJArzBOVanl_Vz0/edit?usp=sharing